### PR TITLE
Added support for [customPostAuthPage]

### PR DESCRIPTION
**Pull request by kirill-21**
This pull request adds support for custom post auth html page

### CODE REVIEW

File: googleapis_auth/lib/auth_io.dart [hunk 1]
```diff
@@ -115,13 +115,16 @@ Future<AutoRefreshingAuthClient> clientViaApplicationDefaultCredentials({
 /// {@macro googleapis_auth_close_the_client}
 /// {@macro googleapis_auth_not_close_the_baseClient}
 /// {@macro googleapis_auth_listen_port}
+///
+/// {@macro googleapis_auth_custom_post_auth_page}
```
Comment (kevmoo) [inline googleapis_auth/lib/auth_io.dart:119]: You need to define this macro somewhere!

**Review by kevmoo — CHANGES_REQUESTED** [review]

File: googleapis_auth/lib/auth_io.dart (hunk 1 above)
Comment (kirill-21) [reply googleapis_auth/lib/auth_io.dart:119]: And where do I do that?

Comment (kirill-21) [reply googleapis_auth/lib/auth_io.dart:119]: > You need to define this macro somewhere!

I just never done it before

**Comment by kirill-21** [comment]
@kevmoo Hi, can you review it again? I've added changes you requested for

**Comment by kevmoo** [comment]
I'm SLAMMED busy this week. Will try to review soon, though!

**Comment by kirill-21** [comment]
> I'm SLAMMED busy this week. Will try to review soon, though!

Okay, waiting

**Comment by kirill-21** [comment]
@kevmoo still no check?

**Comment by kirill-21** [comment]
@kevmoo, hi, when will you able to check and merge this pull request?

**Comment by kirill-21** [comment]
@kevmoo approve pls?

File: googleapis_auth/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ## 1.5.1
 
 - `auth_browser.dart`: handle pop-up closed errors correctly. 
+- Added support for custom post-auth html page
```
Comment (kevmoo) [inline googleapis_auth/CHANGELOG.md:4]: Add a 1.6.0 section above and add this. Update the pubspec accordingly!

**Review by kevmoo — CHANGES_REQUESTED** [review]

File: googleapis_auth/CHANGELOG.md (hunk 2 above)
Comment (kirill-21) [reply googleapis_auth/CHANGELOG.md]: > Add a 1.6.0 section above and add this. Update the pubspec accordingly!

Done

**Review by kevmoo — APPROVED** [review]

**Comment by kevmoo** [comment]
published! https://pub.dev/packages/googleapis_auth/versions/1.6.0

**Pull request merged**